Description: Disable python makefile, this is handled by dh_python* instead.
Author: Roger Light <roger@atchoo.org>
Forwarded: not-needed
--- a/lib/python/Makefile
+++ b/lib/python/Makefile
@@ -5,13 +5,13 @@
 
 .PHONY : all clean install
 
-all : mosquitto.pyc
+all : 
 
 install : all
-	python ./setup.py install --prefix=${prefix} --root=${DESTDIR}
+	#python ./setup.py install --prefix=${prefix} --root=${DESTDIR}
 
 mosquitto.pyc : mosquitto.py
-	python ./setup.py build
+	#python ./setup.py build
 
 clean :
 	-rm -rf build mosquitto.pyc __pycache__
